lib/helpers.js

```diff
@@ -524,3 +545,7 @@ exports.getBluemixDefaultPlan = function(serviceType) {
     default:                    return 'Lite';
   }
 }
+
+exports.ignoreFile = function(ignoreLine) {
```

Comment:
@tunniclm What do you mean exactly? Looks like `fs` is used everywhere in the generator.